planner/core/optimizer.go

```diff
@@ -78,10 +79,12 @@ var optRuleList = []logicalOptRule{
 	&projectionEliminator{},
 	&maxMinEliminator{},
 	&ppdSolver{},
+	&collectPredicateColumnsPoint{},
```

Comment:
@time-and-fate please help confirm the position of collect columns and sync-wait.